docs/components/clients/users.ts

```diff
@@ -0,0 +1,338 @@
+    pinned: true,
+  },
+  {
+    caption: "Groupon",
+    image: "/images/logos/groupon.svg",
+    infoLink: "https://groupon.com",
+    pinned: true,
+    style: {
+      width: 125,
+    },
+  },
+  {
+    caption: "GoodRx",
+    image: "/images/logos/goodrx.svg",
+    infoLink: "https://goodrx.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "Tripadvisor",
+    image: "/images/logos/tripadvisor.svg",
+    infoLink: "https://tripadvisor.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "RapidAPI",
+    image: "/images/logos/rapidapi.svg",
+    infoLink: "https://rapidapi.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "Miro",
+    image: "/images/logos/miro.svg",
+    infoLink: "https://miro.com",
+    pinned: true,
+  },
+  {
+    caption: "Lattice",
+    image: "/images/logos/lattice.svg",
+    infoLink: "https://lattice.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "Watershed",
+    image: "/images/logos/watershed.svg",
+    infoLink: "https://watershed.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+
+  {
+    caption: "Stedi",
+    image: "/images/logos/stedi.svg",
+    infoLink: "https://stedi.com",
+    pinned: true,
+    style: {
+      width: 75,
+    },
+  },
+  {
+    caption: "Framer",
+    image: "/images/logos/framer.svg",
+    infoLink: "https://framer.com",
+    pinned: true,
+  },
+  {
+    caption: "Contentful",
+    image: "/images/logos/contentful.svg",
+    infoLink: "https://contentful.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "Cal.com",
+    image: "/images/logos/calcom.svg",
+    infoLink: "https://cal.com",
+    pinned: true,
+  },
+  {
+    caption: "Codesandbox",
+    image: "/images/logos/codesandbox.svg",
+    infoLink: "https://codesandbox.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "WooCommerce",
+    image: "/images/logos/woocommerce.svg",
+    infoLink: "https://woocommerce.com",
+    pinned: true,
+    style: {
+      width: 150,
+    },
+  },
+  {
+    caption: "Expo",
+    image: "/images/logos/expo.svg",
+    infoLink: "https://expo.io",
+    pinned: true,
+  },
+  {
+    caption: "TeeSpring",
+    image: "/images/logos/teespring.svg",
+    infoLink: "https://spri.ng",
+    pinned: true,
+  },
+  {
+    caption: "Endear",
+    image: "/images/logos/endear.svg",
+    infoLink: "https://endearhq.com",
+    pinned: true,
+  },
+  {
+    caption: "Makeswift",
+    image: "/images/logos/makeswift.svg",
+    infoLink: "https://makeswift.com",
+    pinned: true,
+  },
+  {
+    caption: "Fandom",
+    image: "/images/logos/fandom.svg",
+    infoLink: "https://fandom.com",
+    pinned: true,
+  },
+  {
+    caption: "Waggel",
+    image: "/images/logos/waggel.svg",
+    infoLink: "https://waggel.com",
```

Comment:
```suggestion
    infoLink: "https://waggel.co.uk",
```
I believe this is the correct site here.